Expands aux base construction to include a place for building shuttles

nmajask opened the pull request:
# Document the changes in your pull request

Expands Aux Base Construction to include a place to build shuttles. It has also been merged with port bow solars, so I had to mess with maints a bit to get it to work. The mining shuttle also docks to the open area instead of inside the base construction area. The amount of wires needed to wire the solar panels is still 40, tough it is a bit of a walk. There is also some supplies for building shuttles in a storage room next to the solars room. I have not tested this, so some things may break because of the scale of the changes and my lack of experience.

## Images
### New Aux Base Construction
![image](https://user-images.githubusercontent.com/14363906/137433634-b3bbf2a1-d3f3-4eb7-8133-07b9c661172f.png)

![image](https://user-images.githubusercontent.com/14363906/137433968-7d5af9b1-074f-4fd1-bb14-b0aadd6a0c1f.png)

![image](https://user-images.githubusercontent.com/14363906/137434067-2c7c062f-4ebb-4b1e-96b6-81098302817d.png)

### Starting Boards
![image](https://user-images.githubusercontent.com/14363906/137433734-815a79e9-4d37-48bc-bf73-41f800b30a93.png)
![image](https://user-images.githubusercontent.com/14363906/137433879-7af672c2-7e6e-4b3f-9c09-4d7d5ef37244.png)

# Wiki Documentation

Many images will need to be changed, and if there is a page for aux base construction that will need to be updated too. 

# Changelog

:cl:  
tweak: expanded aux base construction to include a place for building shuttles
/:cl:

nmajask committed:
Expanded aux base construction to include a place for building shuttles

nmajask committed:
I do as the map man says

nmajask committed:
Adds a room for aux base construction

Adds a room for aux base construction, may or may not work.

patpol4 reviewed (changes requested):
1. Are these all just fuel ports? also check the indoor one, also they need dirs to connect to the pipes
![image](https://user-images.githubusercontent.com/48154165/138512175-f4de168e-8f7c-4329-b76e-df4ef228c109.png)
2.  is this supposed to cut off like that? seems weird, either do a corner or attach something to them
![image](https://user-images.githubusercontent.com/48154165/138512283-0a7f190b-e30d-407e-bf9a-e91a328daecc.png)
3. Some of the solar turfs have double cables on them, double check please
![image](https://user-images.githubusercontent.com/48154165/138512380-14101494-ff29-4ded-93c3-18c5ebc5431c.png)
4. The stripe decals won't be visible on catwalks so remove those
5. double decal and missing firelocks here, also remember to add them on the shutter and on the red airlock to the right
![image](https://user-images.githubusercontent.com/48154165/138512729-745ef77c-a96e-440c-8f59-ca78a2780b32.png)
6. this isn't engineering secure storage
![image](https://user-images.githubusercontent.com/48154165/138513051-01b13346-0ec4-4265-9481-deeab5b48f48.png)
7. This is an oversight right?
![image](https://user-images.githubusercontent.com/48154165/138513346-a67447ff-0feb-4910-a4ca-021807c09c2c.png)

I think this is it but i might have missed some, the rest will come out after merge probably

nmajask committed:
Fixes what the map man said

nmajask committed:
Changes custom shuttles and mining shuttles so they can dock to aux

Changes custom shuttles and mining shuttles so they can dock to aux base construction.

patpol4 reviewed (approved):
Yes, sorry I meant left
